link_test.go

```diff
@@ -664,3 +667,140 @@ func TestLinkStats64UnmarshalBinary(t *testing.T) {
 		})
 	}
 }
+
+func TestLinkXDP(t *testing.T) {
```

Comment:
Could you move this test to a `link_live_test.go` file and add a `// +build integration` tag?
That way it will only test on linux and skip on macos.